src/colvarcomp_torchann.cpp

```diff
@@ -0,0 +1,237 @@
+// -*- c++ -*-
+
+// This file is part of the Collective Variables module (Colvars).
+// The original version of Colvars and its updates are located at:
+// https://github.com/Colvars/colvars
+// Please update all Colvars source files before making any changes.
+// If you wish to distribute your changes, please submit them to the
+// Colvars repository at GitHub.
+
+#ifdef TORCH
+
+#include "colvarmodule.h"
+#include "colvar.h"
+#include "colvarvalue.h"
+#include "colvarparse.h"
+#include "colvarcomp.h"
+
+colvar::torchANN::torchANN(std::string const &conf): linearCombination(conf) {
+  set_function_type("torchANN");
+
+  x.type(colvarvalue::type_scalar);
+  enable(f_cvc_scalar);
+
+  if (period != 0.0) {
+    enable(f_cvc_periodic);
+  }
+
+  if ((wrap_center != 0.0) && !is_enabled(f_cvc_periodic)) {
+    cvm::error("Error: wrapAround was defined in a torchANN component,"
+                " but its period has not been set.\n");
+    return;
+  }
+
+  std::string model_file ;
+  get_keyval(conf, "modelFile", model_file, std::string(""));
+  try {
+    nn = torch::jit::load(model_file);
```

Comment:
Sometimes people train the model on GPU and save it, and I guess if we load the model as is the code will fail. Is it better to add a line:
```
nn.to(at::Device("cpu"));
```
to make the code foolproof?